garminworkouts/models/power.py

```diff
@@ -32,4 +32,4 @@ def _has_percent(self):
 
     @staticmethod
     def _to_absolute(power, ftp):
-        return int(power) * ftp / 100
+        return float(power) * ftp / 100
```

Comment:
What is the reason of this change?

Reply:
not sure why it should be int :-) 
solved in 5999552